### PR TITLE
Docs for Snappy CSV format

### PR DESCRIPTION
**Pull request by olpie101**
based on issue #18

### CODE REVIEW

File: docs/csv_format.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  en,Appl. Results,Successful,"If your application is approved, you'll get a Section 24 permit (refugee status). Your permit is valid for 2 to 4 years. You must renew it at your RRO before it expires. Refugees can apply for a refugee ID (maroon ‘ID') and travel documents. It may take time. Only apply for travel documents if you have a refugee ID. If you don't, and must travel for an emergency, contact UNHCR. Tip: If you travel back to your country, you could lose your refugee status in SA."
+  en,Appl. Results,Unsuccessful,"If a RRO rejects your application, you're not recognised as a refugee. You have 30 days to give a notice of appeal, otherwise you must document your stay in another way or leave SA. The reason for rejection affects your appeal. If your application. was fraudulent or abusive, the Standing Committee of Refugee Affairs will review it. You have 14 days to give them a written statement on why you disagree, at the RRO that issued the rejection letter. Ask a LHR counsellor for help."
+
+The first line must contain all the column headers in order to identify which
+column is which. Each following line is provides and entry for the given topic.
```
Comment (hodgestar) [inline docs/csv_format.rst]: s/is provides and entry for the given topic/provides an entry for a topic/.

File: docs/csv_format.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+In snappy topics belong to a particular FAQ. The above example would add a
```
Comment (hodgestar) [inline docs/csv_format.rst]: s/In snappy/In snappy,/
s/add a/add/

File: docs/csv_format.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+2 topics and 5 questions. For example if the FAQ name was *Step1: Applying for Asylum*,
+the following would produce the following structure:
```
Comment (hodgestar) [inline docs/csv_format.rst]: s/the following would/the example would/

File: docs/csv_format.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+        * Answer
+
+      * [en] Unsuccessful
+
+        * Answer
+
+.. warning::
+
+   Make sure to surround any entries which contain commas in quotes (ie. "Testing, Testing, Testing.").
```
Comment (hodgestar) [inline docs/csv_format.rst]: s/in quotes/with quotes/

**Comment by hodgestar** [comment]
Left a few comments, but looks good!

**Comment by hodgestar** [comment]
We should also get these up on readthedocs.org.

**Comment by olpie101** [comment]
Fixed the typos you found

File: docs/csv_format.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  en,Appl. Results,Successful,"If your application is approved, you'll get a Section 24 permit (refugee status). Your permit is valid for 2 to 4 years. You must renew it at your RRO before it expires. Refugees can apply for a refugee ID (maroon ‘ID') and travel documents. It may take time. Only apply for travel documents if you have a refugee ID. If you don't, and must travel for an emergency, contact UNHCR. Tip: If you travel back to your country, you could lose your refugee status in SA."
+  en,Appl. Results,Unsuccessful,"If a RRO rejects your application, you're not recognised as a refugee. You have 30 days to give a notice of appeal, otherwise you must document your stay in another way or leave SA. The reason for rejection affects your appeal. If your application. was fraudulent or abusive, the Standing Committee of Refugee Affairs will review it. You have 14 days to give them a written statement on why you disagree, at the RRO that issued the rejection letter. Ask a LHR counsellor for help."
+
+The first line must contain all the column headers in order to identify which
+column is which. Each of the following lines provides and entry a topic.
```
Comment (hodgestar) [inline docs/csv_format.rst]: s/provides and entry a topic/specifies a question in the FAQ/

**Comment by hodgestar** [comment]
Left one more comment. Otherwise looks good.

**Comment by hodgestar** [comment]
:+1: (slightly retroactive)

**Pull request merged**